src/BugsnagUnity/Native/Android/NativeStackFrame.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using BugsnagUnity.Payload;
+using UnityEngine;
+
+namespace BugsnagUnity
+{
+    internal class NativeStackFrame : NativePayloadClassWrapper, IStackframe
+    {
+        public NativeStackFrame(AndroidJavaObject androidJavaObject) : base (androidJavaObject){}
+
+        public string FrameAddress { get; set; }
+        public string IsLr { get; set; }
+        public string IsPc { get; set; }
```

Comment:
isLr/isPC should be booleans I believe (the docs aren't great on this)

Reply:
They are booleans on both platforms.

Reply:
Fixed